code/modules/mob/living/simple_animal/hostile/mining/goliath.dm

```diff
@@ -119,6 +130,7 @@
 	var/list/cached_tentacle_turfs
 	var/turf/last_location
 	var/tentacle_recheck_cooldown = 100
+	var/juvgoliathsspawned = 0
```

Comment:
nitpick, but looking at code this var actually tracks both spawned and adopted juvenile goliaths, so it isn't accurately named

Reply:
Changed to "juvgoliaths_owned" in the next commit.